### DISCUSSION TITLE
mpv player livelyproperty null

### DISCUSSION THREAD

**Discussion by gnsilence** [rocksdanister/lively#1363] [Q&A]
![image](https://user-images.githubusercontent.com/18021366/186872868-f5cb77d3-dc8e-46b9-9adb-4a33d40c611f.png)
The above error message appears when I run the project.
Do I need to configure anything extra?

**Comment by rocksdanister** [comment]
Not required, if you want customization create the [LivelyProperties.json](https://github.com/rocksdanister/lively/wiki/Video-Guide#mpv) file [here](https://github.com/rocksdanister/lively/blob/29016c245406cbacf9865e4503d29299ad5b6191/src/Lively/Lively.Common/Constants.cs#L20)

Example:
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper\Mpv\LivelyProperties.json`

<img width="290" alt="image" src="https://user-images.githubusercontent.com/17554161/186875666-70b12bd8-cb7c-4c51-88f4-abf7fb91c599.png">

**Reply by gnsilence** [reply]
And it does work,But I can't set the wallpaper.
The exception information is as follows:

Lively.Core.Wallpapers.VideoMpvPlayer::Mpv4:  (+) Video --vid=1 (mjpeg 3440x1440 1.000fps)
Lively.Core.Wallpapers.VideoMpvPlayer::Mpv4: VO: [gpu] 3440x1440 yuv444p
Lively.Core.Watchdog.WatchdogProcess::Cleared watchdog program(s)..
Lively.Core.WinDesktopCore::Sending wallpaper(Screen): \\.\DISPLAY1 | {X=0,Y=0,Width=4096,Height=2160}
Lively.Core.Watchdog.WatchdogProcess::Adding program to watchdog: 10160
Exceptions raised "System.ObjectDisposedException" Located in the "System.IO.Pipes.dll"

**Reply by rocksdanister** [reply]
Share the logfile.

**Reply by gnsilence** [reply]
> Share the logfile.

![image](https://user-images.githubusercontent.com/18021366/186886461-78869170-f888-401d-9d34-ca4ac08ca197.png)
I found that if I clicked on the Set WALLPAPER and then clicked on the CLOSE button, the wallpaper switch would take effect

**Reply by gnsilence** [reply]
> Share the logfile.

The first error that comes up every time is:
Exception raised: "System.ArgumentException" (located in lively.common.dll)

**Reply by rocksdanister** [reply]
Run the installer and not the project -  Create a new issue with a sample video showing the issue and `Settings -> System -> Logfile` file attached.
